topo/node/alpine/alpine_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package alpine
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/h-fam/errdiff"
+	tpb "github.com/openconfig/kne/proto/topo"
+	"github.com/openconfig/kne/topo/node"
+	"google.golang.org/protobuf/encoding/prototext"
+	"google.golang.org/protobuf/proto"
+)
+
+func TestNew(t *testing.T) {
```

Comment:
In general our unit test coverage of node impls is lacking, however going forward this is something we are more actively trying to improve

can you add a test for CreatePod using a fake k8 client? (similar to https://github.com/openconfig/kne/blob/303b49d439a6c31fda957b6d5b8b44898da2908b/topo/node/node_test.go#L180)

Reply:
Done